platform-hub-web/src/app/kubernetes-tokens/kubernetes-tokens-list.html

```diff
@@ -0,0 +1,100 @@
+  <md-toolbar md-scroll-shrink>
+    <div class="md-toolbar-tools">
+      <h3>
+        <span>Kubernetes Tokens</span>
+        <span ng-if="$ctrl.user">for {{$ctrl.user.name}} ({{$ctrl.user.email}})</span>
+      </h3>
+      <span flex></span>
+      <md-button
+        ng-if="$ctrl.isAdmin && $ctrl.user"
+        aria-label="Add new token"
+        ui-sref="kubernetes-tokens.new({userId: $ctrl.user.id})">
+        <md-icon>add_box</md-icon>
+        New
+      </md-button>
+    </div>
+  </md-toolbar>
+
+  <loading-indicator loading="$ctrl.loading"></loading-indicator>
+
+  <md-toolbar md-colors="{background: 'primary-50'}" ng-if="$ctrl.isAdmin || $ctrl.isProjectManager">
+    <div class="md-toolbar-tools">
+      <md-autocomplete
+        md-no-cache="true"
+        md-selected-item="$ctrl.searchSelectedUser"
+        md-search-text="$ctrl.searchText"
+        md-items="user in $ctrl.searchUsers($ctrl.searchText)"
+        md-item-text="user.name"
+        md-min-length="1"
+        ng-model-options="{ debounce: 500 }"
+        placeholder="Filter by user name"
+        flex>
+        <md-item-template>
+          <span md-highlight-text="$ctrl.searchText" md-highlight-flags="^i">
+            {{user.name}}
+            ({{user.email}})
+          </span>
+        </md-item-template>
+        <md-not-found>
+          No users matching "{{$ctrl.searchText}}" were found.
+        </md-not-found>
+      </md-autocomplete>
+      <md-button
+        class="md-primary md-raised"
+        aria-label="Filter list by user"
+        ng-click="$ctrl.filterKubernetesTokensByUser()"
+        ng-disabled="!$ctrl.searchSelectedUser">
+        Filter
+      </md-button>
+    </div>
+  </md-toolbar>
+
+  <md-content layout-padding ng-if="!$ctrl.loading && $ctrl.tokens.length > 0">
+
+    <md-card ng-repeat="t in $ctrl.tokens track by t.cluster">
+      <md-card-title>
+        <md-card-title-text>
+          <span class="md-headline">
+            {{t.cluster}} cluster
+          </span>
+          <p>
+            <strong>Token:</strong> {{t.token}}
+          </p>
+          <p>
+            <strong>Groups:</strong> {{t.groups.join(', ')}}
+          </p>
+        </md-card-title-text>
+      </md-card-title>
+
+      <md-card-actions layout="row" layout-align="start center">
+        <md-button
+          class="md-primary"
+          aria-label="View and edit details for this token"
+          ui-sref="kubernetes-tokens.edit({userId: t.user.id, cluster: t.cluster})">
+          Edit
+        </md-button>
+        <md-button
+          class="md-primary"
+          ng-click="$ctrl.deleteToken(t.user.id, t.cluster, $event)"
+          aria-label="Remove kubernetes token">
+          Delete
+        </md-button>
+      </md-card-actions>
+    </md-card>
+
+  </md-content>
+
+  <md-content layout-padding ng-if="!$ctrl.loading && $ctrl.tokens.length == 0">
+    <md-card>
+      <md-card-title>
+        <md-card-title-text>
+          <span>
```

Comment:
If you add the class `none-text` here (something specific to our codebase) then it will render the text in a way which is consistent with other places where no results/items are found.